ab3d2_source/c/system.c

```diff
@@ -360,6 +362,11 @@ void Sys_FrameLap()
 
 void Sys_EvalFPS()
 {
+#ifndef DEV
```

Comment:
I just _KNOW_ people will want to be able to see FPS, even in release mode.
TKG could become a benchmarking tool for people.